doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,325 @@
+.. _ncs_release_notes_100:
+
+|NCS| v1.0.0 Release Notes
+##########################
+
+nRF Connect SDK delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products. It includes the MCUboot and the Zephyr RTOS open source projects which are continuously integrated and re-distributed with the SDK.
+
+nRF Connect SDK v1.0.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the nRF Connect SDK.
+
+Highlights
+**********
+
+* |NCS| v1.0.0 is aligned with the production release of nRF9160.
+* Added a new AWS FOTA sample demonstrating application firmware update through the use of AWS IoT Jobs with MQTT and HTTP.
+* Added secure services in Secure Partition Manager (SPM) for TrustZone devices (nRF9160).
+* Updated the Download Client library with support for HTTPS and IPv6.
+* Included a new version of the nRF BLE Controller.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.0.0**.
+Check the ``west.yml`` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`Getting the nRF Connect SDK code <cloning_the_repositories_win>` for more information.
+
+Supported modem firmware:
+*******************
+* mfw_nrf9160_1.0.0
+
+Supported boards
+****************
+
+* PCA10090 (nRF9160 DK)
+* PCA10056 (nRF52840 Development Kit)
+* PCA10059 (nRF52840 Dongle)
+* PCA10040 (nRF52 Development Kit)
+* PCA10028 (nRF51 Development Kit)
+* PCA63519 (Smart Remote 3 DK add-on)
+
+
+Required tools
+**************
+
+In addition to the tools mentioned in :ref:`gs_installing`, the following tool versions are required to work with the |NCS|:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Tool
+     - Version
+     - Download link
+   * - SEGGER J-Link
+     - V6.41
+     - `J-Link Software and Documentation Pack`_
+   * - nRF5x Command Line Tools
+     - v9.8.1
```

Comment:
Should also specify :  nRF Connect for Desktop version greater than v3.0.0

Reply:
added here in the table ; additionally, added a note in the Supported modem firmware section that nRF Connect for Desktop must be now used to update modem firmware

Reply:
nRF Connect for Desktop v3.0.0 requires nRF5x Command Line Tools v10.2.0, so might want to change required version from v9.8.1 to v10.2.0

Reply:
All test servers and testing was done with 9.8.1, Hence I recommend not to change .